platform-dom/src/main/java/com/softicar/platform/dom/elements/testing/engine/IDomTestExecutionEngine.java

```diff
@@ -58,14 +58,15 @@ public interface IDomTestExecutionEngine extends TestRule {
 
 	/**
 	 * Inputs the specified text into the given {@link IDomTextualInput} and
-	 * triggers an {@link IDomEvent} of type {@link DomEventType#TAB}.
+	 * triggers an {@link IDomEvent} of type {@link DomEventType#TAB} and
```

Comment:
"an event" ... of two distinct types? I am confused: Does it trigger two events, or is that XOR? Should be clarified.

Reply:
fixed